services/document-manager/backend/.env-example

```diff
@@ -5,7 +5,7 @@ DB_PORT=5432
 DB_NAME=mds
 DB_NAME_TEST=mds_test
 
-DOCUMENT_MANAGER_URL=http://localhost:5001
+DOCUMENT_MANAGER_URL=http://document_manager_backend:5001
```

Comment:
This will only work with Docker. Because of the way bridge networking works, with docker it should be able to work with both `localhost` and `document_manager_backend`, but if someone decides to run the document manager locally - say for debugging - this will no longer work.

Was this changed because it no longer worked as intended?

Reply:
I had a note in my PR comments about needing to change an .env file for it to work locally.
We had to change it from localhost to use the explicit hostname because of resource resolution issues. The docman will say "come get the PDF at localhost" which the API will fail to do, because, to itself, it is localhost. Here's the error that bubbles up without the modifications to the .env file:

```{"status":500,"message":"HTTPConnectionPool(host='localhost', port=5001): Max retries exceeded with url:  (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5d49465a20>: Failed to establish a new connection: [Errno 111] Connection refused',))"}```

Reply:
Might be able to get around it with the loopback address 127.0.0.1 or the default 0.0.0.0 rather than localhost.

Reply:
That gives the same error because, to the API, it is also 127.0.0.1/0.0.0.0.

Reply:
Read Line 85 of the document.py resource in the document manager. It sets the response header and tells the client where they can continue uploading the file. So the backend pod tries to send the the data to the document manager via 'localhost' but localhost:5001 doesn't exist on the backend pod, so we need to use the service name. this will not be an issue in openshift because it's always uses the service name.

Reply:
Yeah, but the document_manager should be able to bind to the loopback address across all the containers, which is why I'm confused.

But @exzizt walked me through the implementation and it does seem like blocking behaviour, so I'm alright with it.